src/components/Tabs/Tabs.js

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from "react";
+
+import TabNavItems from "./TabNavItems";
+import EventItem from "../Events/EventItem";
+
+import "./Tabs.css";
+
+const TabPanel = (props) => {
+  const { children, value, index } = props;
+
+  return (
+    <div hidden={value !== index}>
+      {value === index && <div>{children}</div>}
+    </div>
+  );
+};
+
+const Tabs = (props) => {
+  const allyears = props.data.map((item) => new Date(item.date).getFullYear());
+
+  const years = allyears
+    .filter(function (item, position) {
+      return allyears.indexOf(item) === position;
+    })
+    .sort()
+    .reverse();
+
+  const [value, setValue] = useState(0);
+
+  const handleOnClick = (e, newValue) => {
+    setValue(newValue);
+  };
+
+  return (
+    <div className='tabs'>
+      <ul className='nav'>
+        <TabNavItems
+          data={props.data}
```

Comment:
If we pass `years` instead of `props.data`, the logic in `TabNavItems` could be simpler.